fluent-fallback/src/localization.rs

```diff
@@ -16,21 +17,24 @@ where
     generator: G,
     provider: P,
     sync: bool,
-    res_ids: Vec<ResourceId>,
+    res_ids: FxHashSet<ResourceId>,
 }
 
 impl<G, P> Localization<G, P>
 where
     G: BundleGenerator<LocalesIter = P::Iter> + Default,
     P: LocalesProvider + Default,
 {
-    pub fn new(res_ids: Vec<ResourceId>, sync: bool) -> Self {
```

Comment:
I made this method generic rather than requiring the `FxHashSet` to be part of the public API.